src/styles/card.css

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
Use * selector for this

Reply:
If `flex-direction: column` move to * selector I'll need to `add flex-direction: row` into `.stars, .card__rating, .card__price`. 
If `display: flex` move to * selector title `Product cards` from head tag will appear on the page. Because it's default `display: none` changes
What is the sense of this moving? I'll increase my code as a result.

Reply:
I meant just box-sizing